src/Asset/RevisionCompiler.php

```diff
@@ -190,11 +200,15 @@ protected function putRevision($revision)
      */
     public function flush()
     {
-        $revision = $this->getRevision();
+        if (is_subclass_of($this, self::class)) {
```

Comment:
Yikes, I'm sorry, I just can't merge this 😝 

If we want a temporary solution, just slap a hardcoded `unlink($this->getDestination() . '/rev-manifest.json')` in `WebAppAssets::flush` I reckon